README.md

```diff
@@ -97,4 +102,6 @@ The admin of your GCP project/folder/org to complete the steps below.
         -   Disallow force pushes
         -   Disallow deletions
 
-## Adding code other than AOD (TODO #9)
+## Adding code other than AOD
+
+It should be the same as you adding code in your other repositories.
```

Comment:
If the branch is locked, how to add code?

Reply:
Adding more details.

Reply:
But if branch is locked, passing all checks still won't allow you to merge.

Reply:
Ah..forgot the branch is locked. It needs the PR author to ask for bypass branch protections permission. Wouldn't it bypass all protections including require PR, checks and approvals?

Reply:
Maybe we don't need to provide too much detail, so the recommended way is to use [restrict-who-can-push-to-matching-branches](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches#restrict-who-can-push-to-matching-branches) setting to allow a certain group to push to main but still require PR, checks and approval. PR authors can request to be added to that group temporary for merging a code change?